libV2/tasks/BaseTask.js

```diff
@@ -104,6 +94,22 @@ class BaseTask extends Process {
     async _join() {
         return this._cache.disconnect();
     }
+
+    withWarp10(func, onError) {
+        return iterIfError(this._warp10Clients, func, error => {
+            const {
+                name, code, message, stack,
+            } = error;
+            logger.error('error during warp 10 request', {
```

Comment:
Thinking out loud if all errors should be logged in error, or if some errors are "normal" and should be logged in debug/info

Reply:
All cases I can think of meet the "error" qualification, though I can see value in being able to silence this. What do you think of changing it to work as a default value for `onError`? That should allow opting out on a case by case basis.